src/parameterizations/vertical/MOM_bkgnd_mixing.F90

```diff
@@ -411,7 +338,11 @@ subroutine calculate_bkgnd_mixing(h, tv, N2_lay, Kd_lay, Kv, j, G, GV, US, CS)
 
   ! set some parameters
   deg_to_rad = atan(1.0)/45.0 ! = PI/180
-  epsilon = 1.e-10
+  min_sinlat = 1.e-10
```

Comment:
Curiously, if you are willing to work in radians instead of degrees, this value of the minimum value of sin(lat) is also the minimum value of lat to within 64-bit machine precision, since `sin(x) = x * (1 - x^2/3! + x^4/5! - ...)` and (1.e-10)**2/6 < 1.0e-17. So maybe we just need to revise the description in the comment (or not).